.vib/ghost/cypress/cypress/integration/ghost_spec.js

```diff
@@ -0,0 +1,123 @@
+/// <reference types="cypress" />
+import { random, getPageUrlFromTitle, getUserFromEmail } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login(Cypress.env('email'), Cypress.env('password'), false);
```

Comment:
Could we always have this set the 'waitForDashboard' to `true`?
We are waiting for it in the login test and in the `commands.js` default settings. 
Better said, In which case will we will **not** wait for the dashboard?

Reply:
> Better said, In which case will we will not wait for the dashboard?

This is the only case in which we **can't wait for it**. Upon submitting the login form, Ghost will do different redirections and eventually show the Dashboard page. It is not only until those last redirections happen that one can navigate to other pages, else you are presented with an Unauthorized error. In short, this is the reason why I decided to add it to commands.

Why don't we want to wait for it here? As we are validating the log-in feature, I want to explicitly validate that there are no form errors. If I wait for Dashboard to appear, it means that the log-in form is already gone.

I don't know if this is kind of tricky and I should just always validate that in `commands` and that's it. WDYT?